wbooks/src/interfaces/Book.ts

```diff
@@ -7,3 +7,8 @@ export interface Book {
   year: string;
   imageUrl?: string | null;
 }
+
+export interface BookState {
+  books: Book[];
+  book?: Book;
```

Comment:
Why this?

Reply:
I need to know what's the correct type for book state